package.json

```diff
@@ -1,8 +1,9 @@
 {
-  "name": "my-app",
+  "name": "andromeda",
```

Comment:
About time somebody had the balls to change this

Reply:
Ahahaha

Reply:
I could have put andromeda